lib/index.js

```diff
@@ -0,0 +1,96 @@
+const config = require('./config');
+
+var request = require('request-promise-native');
+
+
+module.exports = function(apikey) {
+  this.key = apikey;
+
+  this.samlEntity = function(name, entityId, federationId) {
```

Comment:
I made multiple exports from a single module so on that the client is separated by the entity creation, but also both can be imported with a single import/require by the client application.